docs/topics/proc_interacting-with-the-secured-booster-on-the-command-line.adoc

```diff
@@ -41,6 +41,8 @@ $ curl -sk -X POST https://<SSO_AUTH_SERVER_URL>/auth/realms/$REALM/protocol/ope
 ----
 +
 NOTE: The `-sk` option tells curl to ignore failures resulting from self-signed certificates. Do not use this option in a production environment.
++
+IMPORTANT: On macOS, you must have `curl` version `7.56.1` or greater installed.
```

Comment:
I would put the information into the note above (the MInimalism initiative commented that our docs have too many admonitions, especially close together, which makes reading hard). Also, consider adding the information @metacosm noted about building curl with OpenSSL.